### PR TITLE
Added QFN-16-1EP_4x4mm_P0.65mm_EP2.5x2.5mm

### PR DESCRIPTION
**Pull request by JeppeSRC**
Added QFN-16-1EP_4x4mm_P0.65mm_EP2.5x2.5mm

### CODE REVIEW

**Comment by poeschlr** [comment]
This datasheet contains quite a few package specifications. could you directly link to the page the used specification is on? syntax: [url]#page=[pdf page number]

**Comment by JeppeSRC** [comment]
[Datasheet](http://ww1.microchip.com/downloads/en/PackagingSpec/00000049BQ.pdf#page=266)

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-1x.yaml [hunk 1]
```diff
@@ -420,6 +420,59 @@ QFN-16-1EP_4x4mm_P0.5mm_EP2.45x2.45mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+QFN-16-1EP_4x4mm_P0.65mm_EP2.5x2.5mm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://ww1.microchip.com/downloads/en/PackagingSpec/00000049BQ.pdf'
```
Comment (poeschlr) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-1x.yaml]: point directly to the correct page here as well: http://ww1.microchip.com/downloads/en/PackagingSpec/00000049BQ.pdf#page=266

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-1x.yaml [hunk 2]
```diff
@@ -420,6 +420,59 @@ QFN-16-1EP_4x4mm_P0.5mm_EP2.45x2.45mm:
...
+  lead_width:
+    minimum: 0.25
+    nominal: 0.30
+    maximum: 0.35
+  lead_len:
+    minimum: 0.30
+    nominal: 0.40
+    maximum: 0.50
+
+  EP_size_x: 2.5
```
Comment (poeschlr) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-1x.yaml]: Same as with your other pull request, set this to the package ep size dimensions (including tolerances) and use the overwrite parameter if necessary.

**Comment by cpresser** [comment]
Question: Why do we allow `EP_size_x_override`?

KLC says this:
image

**Comment by poeschlr** [comment]
@chschlue in this case the ep size is equal to minimum so the KLC rule is fulfilled. (The rule was written with knowledge of this one document.)

**Comment by poeschlr** [comment]
thanks

**Comment by chschlue** [comment]
@poeschlr ?

**Pull request merged**